kotlin/src/main/com/looker/sdk/4.0/methods.kt

```diff
@@ -48,12 +50,13 @@ class LookerSDK(authSession: AuthSession) : APIMethods(authSession) {
      * POST /alerts/{alert_id}/follow -> Void
      */
     fun follow_alert(
-        alert_id: String,
-    ): SDKResponse {
+        alert_id: String
```

Comment:
@tjbanghart are we missing a lint/reformatting update for Kotlin as part of our generation process?

Reply:
Yes, I can add this in a follow up as well

Reply:
@tjbanghart do you want to merge that in here or wait until after this goes through?

cc @jkaster

Reply:
Just ran the linter - all good for now 👍